src/hooks/useCreateRoom.ts

```diff
@@ -58,6 +60,11 @@ export default function useCreateRoom(): ({
                 description: JSON.stringify(description),
             })
 
+            if (storageEnabled) {
```

Comment:
I guess we don't have UI to switch `storageEnabled` off, right?

Reply:
For now it can only be visually edited upon room creation. Could add an item to the room's dropdown that went "Enable/Disable Room Storage" and that could achieve this dynamic assignation of storage